bin/rails

```diff
@@ -1,14 +1,10 @@
 #!/usr/bin/env ruby
-# frozen_string_literal: true
-
 begin
-  load File.expand_path('spring', __dir__)
+  load File.expand_path('../spring', __FILE__)
 rescue LoadError => e
   raise unless e.message.include?('spring')
 end
-
 APP_PATH = File.expand_path('../config/application', __dir__)
 $LOAD_PATH.unshift(File.expand_path('../lib', __dir__))
```

Comment:
This line was manually added (or left as-is in this case).  The fact that this is necessary makes me think we should revisit our autoloading logic.